libraries/rodeos/include/b1/rodeos/callbacks/coverage.hpp

```diff
@@ -16,10 +17,13 @@ class coverage_maps {
 
       cov_map_t funcnt_map;
       cov_map_t linecnt_map;
+      eosio::name T;
    private:
       coverage_maps() = default;
 };
 
+enum class coverage_mode = { func, line };
```

Comment:
Since this is mapped to uint32_t, I would initialize them explicitly.

```
enum class coverage_mode : uint32_t {
   func = 0,
   line = 1
};
```